src/meta-srv/src/service/admin/meta.rs

```diff
@@ -0,0 +1,179 @@
+        let mut key_prefix = String::from(SCHEMA_KEY_PREFIX) + SEPARATOR;
+        match map.get("catalog_name") {
+            Some(catalog_value) => key_prefix = key_prefix + catalog_value + SEPARATOR,
+            None => {
+                return error::MissingRequiredParameterSnafu {
+                    msg: "catalog_name parameter is required".to_string(),
+                }
+                .fail();
+            }
+        }
+        get_key_list_response_by_prefix_key(&key_prefix, &self.kv_store).await
+    }
+}
+
+#[async_trait::async_trait]
+impl HttpHandler for TablesHandler {
+    async fn handle(
+        &self,
+        _: &str,
+        map: &HashMap<String, String>,
+    ) -> Result<http::Response<String>> {
+        let mut key_prefix = String::from(TABLE_KEY_PREFIX) + SEPARATOR;
+        match map.get("catalog_name") {
+            Some(catalog_value) => key_prefix = key_prefix + catalog_value + SEPARATOR,
+            None => {
+                return error::MissingRequiredParameterSnafu {
+                    msg: "catalog_name parameter is required".to_string(),
+                }
+                .fail();
+            }
+        }
+
+        match map.get("schema_name") {
+            Some(schema_value) => key_prefix = key_prefix + schema_value + SEPARATOR,
+            None => {
+                return error::MissingRequiredParameterSnafu {
+                    msg: "schema_name parameter is required".to_string(),
+                }
+                .fail();
+            }
+        }
+        get_key_list_response_by_prefix_key(&key_prefix, &self.kv_store).await
+    }
+}
+
+#[async_trait::async_trait]
+impl HttpHandler for TableHandler {
+    async fn handle(
+        &self,
+        _: &str,
+        map: &HashMap<String, String>,
+    ) -> Result<http::Response<String>> {
+        let mut table_key_prefix = String::from(TABLE_KEY_PREFIX) + SEPARATOR;
+        match map.get("full_table_name") {
+            Some(full_table_name) => {
+                let replace_name = full_table_name.replace('.', SEPARATOR);
+                table_key_prefix = table_key_prefix + &replace_name
+            }
+            None => {
+                return error::MissingRequiredParameterSnafu {
+                    msg: "full_table_name parameter is required".to_string(),
+                }
+                .fail();
+            }
+        }
+
+        let response = self.kv_store.get(table_key_prefix.into_bytes()).await?;
+        let mut value: String = "Not found result".to_string();
+        if let Some(key_value) = response {
+            value = String::from_utf8(key_value.value).context(error::InvalidUtf8ValueSnafu)?;
+        }
+        http::Response::builder()
+            .status(http::StatusCode::OK)
+            .body(value)
+            .context(error::InvalidHttpBodySnafu)
+    }
+}
+
+/// Get kv_store's key list with http response format by prefix key
+async fn get_key_list_response_by_prefix_key(
+    key_prefix: &String,
```

Comment:
Avoid clone of String
```suggestion
    key_prefix: String,
```